plugins/modules/packaging/os/zypper.py

```diff
@@ -511,7 +511,9 @@ def repo_refresh(m):
 
 
 def transactional_updates():
-    return os.path.exists('/var/lib/misc/transactional-update.state')
+    return os.path.exists('/var/lib/misc/transactional-update.state') or \
+           os.path.exists('/sbin/transactional-update') or \
+           os.path.exists('/bin/tukit')
```

Comment:
Another way to handle that could be:
```suggestion
    return any(os.path.exists(p) for p in ['/var/lib/misc/transactional-update.state', '/sbin/transactional-update', '/bin/tukit'])
```